cmd/vm/main_linux.go

```diff
@@ -175,7 +175,7 @@ func rx(conn net.Conn, tap *water.Interface, errCh chan error, mtu int) {
 			log.Info(packet.String())
 		}
 
-		binary.LittleEndian.PutUint16(size, uint16(n))
+		binary.LittleEndian.PutUint16(size, uint16(n)) //nolint: gosec
```

Comment:
This is reminiscent of https://github.com/crc-org/crc/pull/4360#discussion_r1778421899 , it’s better to either add explicit checks to ensure there is no overflow, or to explain why we can ignore the overflow.
You can use `//#nosec G115 ` instead of `//nolint: gosec` to avoid ignoring this altogether.
If you want, you can add an `excludes: G115` for now to golangci-lint `gosec` configuration and fix this in a follow-up PR.